core/ui/PageControls/controlpanel.tid

```diff
@@ -3,6 +3,7 @@ tags: $:/tags/PageControls
 caption: {{$:/core/images/options-button}} {{$:/language/Buttons/ControlPanel/Caption}}
 description: {{$:/language/Buttons/ControlPanel/Hint}}
 
+\whitespace trim
 \define control-panel-button(class)
```

Comment:
`\whitespace trim`

also needs to be inside `\define` otherwise it only works on the tiddler body content.

Reply:
it's sufficient here, it already removes the spaces